### PR TITLE
feat!: make getScreenshot return an actual image

### PR DESCRIPTION
**Pull request by jlipps**
so it's useful for image testing

### CODE REVIEW

File: test/fixtures/app.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<app>
+<app width="667" height="1420">
```
Comment (KazuCocoa) [inline test/fixtures/app.xml:2]: Is it good to reduce the image width/height in order to reduce the file size more?

**Review by KazuCocoa — APPROVED** [review]
just left a minor comment, otherwise LGTM

File: test/fixtures/app.xml (hunk 1 above)
Comment (jlipps) [reply test/fixtures/app.xml:2]: I wanted the image to be high enough resolution for testing image related functionality, it's about 900kb which i understand is a bit. what do you recommend?

Comment (KazuCocoa) [reply test/fixtures/app.xml:2]: I just thought this image could be less size like https://github.com/appium/ruby_lib_core/blob/master/test/functional/data/test_has_blue.png is about 300kb (width 750 × height 1334)
(I usually put an image in 100kb on the web but this is kind of preference:))

File: lib/fake-app.js [hunk 2]
```diff
@@ -123,7 +148,7 @@ class FakeApp {
   }
 
   getScreenshot () {
-    return 'hahahanotreallyascreenshot';
+    return fs.readFileSync(SCREENSHOT, 'base64');
```
Comment (mykola-mokhnach) [inline lib/fake-app.js:151]: cannot we generate the image in runtime using pngjs?

Comment (jlipps) [reply lib/fake-app.js:151]: we could but the main reason i am putting this here is to test image processing stuff against it so it needs to be a known image. i'll see if i can reduce the size.

File: test/fixtures/app.xml (hunk 1 above)
Comment (jlipps) [reply test/fixtures/app.xml]: OK I reduced it to 250kb

File: lib/fake-app.js (hunk 2 above)
Comment (jlipps) [reply lib/fake-app.js:151]: OK i reduced the size

**Pull request merged**